spikingjelly/activation_based/neuron/lif.py

```
@@ -36,15 +36,17 @@ def __init__(
 
         * **中文**
 
-        :class:`LIFNode` 的简化版实现。
+        基于 :class:`SimpleBaseNode` 充电-放电-重置接口的纯 PyTorch LIF 实现，
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)